Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -135,31 +135,31 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         
         if rbaDataToggle.isOn {
             let billingPricing = BTPayPalBillingPricing(
-                pricingModel: .autoReload,
+                pricingModel: .fixed,
                 amount: "9.99",
-                reloadThresholdAmount: "100.00"
+                reloadThresholdAmount: "99.99"
             )
             
             let billingCycle = BTPayPalBillingCycle(
                 interval: .month,
                 intervalCount: 1,
-                numberOfExecutions: 12,
-                sequence: 9,
-                startDate: "2024-04-06T00:00:00Z",
-                isTrial: false,
+                numberOfExecutions: 1,
+                sequence: 1,
+                startDate: "2024-08-01",
```

Comment:
We should change the doc string associated with this input then - I think we are currently telling merchants to use the previous format

Reply:
➕ 1 on updating the doc strings for `startDate`, let also update `billingCycles` to more explicitly call out a max of 3

Reply:
Addressed on this commit: f89d8f94abde78f94eea294ac47d9c3fedb09770

Reply:
@jaxdesmarais Do you mean `billlingCycles` array field in `BTPayPalRecurringBillingDetails` has max of three items? Sorry, I didn't catch context for this.

Reply:
Mentioned max number of` billlingCycles` array items in docStrings in this commit: a27721ec785442b32452343d81a0c4e49ccb7d05